facilitator/src/aggregation.rs

```diff
@@ -246,14 +250,16 @@ impl<'a> BatchAggregator<'a> {
         // Make sure all the parameters in the headers line up
         if !peer_validation_header.check_parameters(&own_validation_header) {
             return Err(anyhow!(
-                "validation headers do not match. Peer: {:?}\nOwn: {:?}",
+                "trace id {} validation headers do not match. Peer: {:?}\nOwn: {:?}",
+                self.trace_id,
                 peer_validation_header,
                 own_validation_header
             ));
         }
         if !ingestion_header.check_parameters(&peer_validation_header) {
             return Err(anyhow!(
-                "ingestion header does not match peer validation header. Ingestion: {:?}\nPeer:{:?}",
+                "trace id {} ingestion header does not match peer validation header. Ingestion: {:?}\nPeer:{:?}",
```

Comment:
This is cool, but it'd be even better if we could log the trace ID as a field in a structured JSON message the way we do on the Go side. If nothing else, it'd make it easier to construct log queries that span `workflow-manager` and `facilitator`. [slog](https://crates.io/crates/slog) seems popular. Maybe we should adopt that?

Reply:
That's in my "TODO" for a future PR that just overhauls logging in facilitator.

Reply:
[tracing](https://crates.io/crates/tracing) also seems popular and is by the Tokio authors, which is promising.

Reply:
>"TODO" for a future PR

Sounds good!